compiler/ballerina-lang/src/main/java/io/ballerina/projects/UsedTypeDefAnalyzer.java

```diff
@@ -0,0 +1,150 @@
+package io.ballerina.projects;
```

Comment:
Missing license header

Reply:
Fixed in c812e4062d376b81a49f24f7fa16fc00a94b7783.